index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Joaquin's Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap" crossorigin="anonymous" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <nav class="flexnav">
+            <a href="index.html"><h3>Welcome</h3></a>
+            <a href="" class="menu">≡</a>
+        </nav>
+    </header>
+    <section class="main">
+        <section class="background"></section>
```

Comment:
- [ ] Congratulations on adding the image as a background, but unfortunately this creates a bug that breaks the entire section, none of the social media icons is clickable and text isn't selectable on both browsers I tested, I think it'd be better if you simply deleted this section with class background, and instead applied the property background and background-size from the background class (the other properties will be unnecessary) to the landing-page class on CSS and see if you wanna adjust anything from there.

Reply:
Thank you for that suggestion, it worked!